src/views/tickets/orders-rsvp.php

```diff
@@ -46,10 +46,12 @@
 				);
 				?>
 			</p>
+			<?php if ( class_exists( 'Tribe__Tickets_Plus__Attendees_List' ) && ! Tribe__Tickets_Plus__Attendees_List::is_hidden_on( get_the_ID() ) ) : ?>
```

Comment:
Rather than doing a `class_exists()` here, I'd suggest adding a `do_action( 'event_tickets_after_user_details', $attendee_group, $post_id );`

Then move the HTML to `event-tickets-plus` as part of an `add_action()`. This should keep the ET+ specific code in ET+